db/schema.rb

```diff
@@ -97,7 +97,7 @@
     t.index ["event_id"], name: "index_tweets_on_event_id"
   end
 
-  create_table "users", id: :serial, force: :cascade do |t|
```

Comment:
why are you deleting the id: :serial?

Reply:
According to https://edgeguides.rubyonrails.org/active_record_migrations.html
>A primary key column called id will also be added implicitly, as it's the default primary key for all Active Record models. 

Btw this was done automatically by Rails when I did a migration. I think this has become a default behaviour since `id`s are sequential and unique, following "convention over config" it only makes sense to get rid of it.